cos-gpu-installer-docker/entrypoint.sh

```
@@ -184,68 +185,63 @@ update_container_ld_cache() {
 download_nvidia_installer() {
   info "Downloading GPU installer ... "
   pushd "${NVIDIA_INSTALL_DIR_CONTAINER}"
-  local -r gpu_installer_download_url="$(get_gpu_installer_url ${NVIDIA_DRIVER_VERSION} ${VERSION_ID} ${BUILD_ID})"
+  local -r gpu_installer_download_url="$(get_gpu_installer_url "${NVIDIA_DRIVER_VERSION}" "${VERSION_ID}" "${BUILD_ID}")"
```

Put the variable declaration on a separate line to avoid masking errors in get_gpu_installer_url

local -r gpu_installer_download_url
gpu_installer_download_url="$(get_gpu_installer_url ...)"

---

Done